### ISSUE TITLE
"Big properties" of objects

### ISSUE THREAD

**Issue by CunningGabe** [MathHubInfo/mhd#49]
I'm working on a dataset where there is a covering relation between some pairs of objects, so I'd like to record which objects cover which. I've computed this, and in my collection of about 20K objects, some objects are covered by more than 1000 of them. So it would be unwieldy to present the user with the full list of 1000 IDs in the database view. However, it would be nice to present the full list to anyone who clicks through to view the item on its own page. 

In short, maybe we want some properties to display differently in the database view vs the item view.
